src/apps/perps/components/mobile/MobilePositionsCard.tsx

```
@@ -0,0 +1,160 @@
+  totalValue: string;
+  totalPnl: string;
+  totalPnlPercent: string;
+  openOrders?: any[];
+}
+
+export function MobilePositionsCard({
+  positions,
+  totalValue,
+  totalPnl,
+  totalPnlPercent,
+  totalPnlPercent,
```

Biome (2.1.2)

[error] Duplicate parameter name.

The parameter overrides a preceding parameter by using the same name.

(lint/suspicious/noDuplicateParameters)